lighthouse-core/gather/gatherers/dobetterweb/doctype.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+/**
+ * Get and return `name`, `publicId`, `systemId` from
+ * `document.doctype`
+ * @return {Object<string>}
```

Comment:
I believe  for the typescript compiler, `Object` needs the key and value types, so it's `Object<string, string>`, though it's only complaining for one of these, so I'm not positive.

Regardless, it would probably be better to do the more specific `@return {{name: string, publicId: string, systemId: string}}` anyways.